loyalty_limit/tests/test_loyalty_limit.py

```diff
@@ -0,0 +1,271 @@
+            {
+                "name": "Test Loyalty Limit Next Order Coupons With Customer Limit",
+                "program_type": "next_order_coupons",
+                "trigger": "auto",
+                "applies_on": "future",
+                "max_customer_application": 2,
+                "rule_ids": [
+                    Command.create(
+                        {
+                            "reward_point_mode": "order",
+                            "minimum_qty": 1,
+                            "minimum_amount": 20,
+                        },
+                    )
+                ],
+                "reward_ids": [
+                    Command.create(
+                        {
+                            "reward_type": "discount",
+                            "required_points": 1,
+                            "discount": 10,
+                            "discount_mode": "percent",
+                            "discount_applicability": "order",
+                        },
+                    )
+                ],
+            }
+        )
+        cls.next_order_coupon_with_salesman_limit = cls.env["loyalty.program"].create(
+            {
+                "name": "Test Loyalty Limit Next Order Coupons With Salesman Limit",
+                "program_type": "next_order_coupons",
+                "trigger": "auto",
+                "applies_on": "future",
+                "salesmen_limit_ids": [
+                    Command.create(
+                        {
+                            "user_id": cls.salesman_1.id,
+                            "max_salesman_application": 2,
+                        },
+                    ),
+                    Command.create(
+                        {
+                            "user_id": cls.salesman_2.id,
+                            "max_salesman_application": 2,
+                        },
+                    ),
+                ],
+                "rule_ids": [
+                    Command.create(
+                        {
+                            "reward_point_mode": "order",
+                            "minimum_qty": 1,
+                            "minimum_amount": 20,
+                        },
+                    )
+                ],
+                "reward_ids": [
+                    Command.create(
+                        {
+                            "reward_type": "discount",
+                            "required_points": 1,
+                            "discount": 10,
+                            "discount_mode": "percent",
+                            "discount_applicability": "order",
+                        },
+                    )
+                ],
+            }
+        )
+
+    def test_salesmen_limit_fields_default_computation(self):
+        """Test that the default computation sets limit counts to 0."""
+        program = self.promotion_with_salesman_limit
+        program._compute_salesmen_limit_count()
```

Comment:
```suggestion
        program._compute_salesmen_limit_count()
```
It is not necessary to call the compute manually, the value should already be appropriate.